docs/guide/core-widgets.md

```diff
@@ -140,9 +140,44 @@ module.exports = {
 }
 ```
 
+### Adding placeholder content
+
+By default, the rich text widget displays default content. To block this behavior, set the `placeholder` option to a value of `false`. To change the content, pass either a simple string or a string containing a namespaced i18n string, to the `placeholderText` option.
```

Comment:
It's not default content, it's placeholder content. The difference is that it's not saved to the database so it can never accidentally become part of the published page.

Reply:
"a namespaced i18n key"

Reply:
Changed for both.